### ISSUE TITLE
chore: assert statements mostly use wrong order

### ISSUE THREAD

**Issue by bobheadxi** [src-d/hercules#246]
according to the [official documentation](https://godoc.org/github.com/stretchr/testify/assert), what you **expect** should come first, and what you **actually get** should come second, for example:

![image](https://user-images.githubusercontent.com/23356519/54487852-0b92c200-4858-11e9-8382-6b0b107cf3ba.png)

so assert statements like:

```go
	assert.Equal(t, tss.tick0.Year(), 1969)
```

should actually be:

```go
	assert.Equal(t, 1969,  tss.tick0.Year())
```

otherwise, you get mildly misleading output:

```
			Error Trace:	ticks_test.go:210
			Error:      	Not equal: 
			            	expected: 1970
			            	actual  : 1969
			Test:       	TestTicksSinceStartConsumeZero
```

^^^ what that should actually say is:

```
			Error Trace:	ticks_test.go:210
			Error:      	Not equal: 
			            	expected: 1969
			            	actual  : 1970
			Test:       	TestTicksSinceStartConsumeZero
```

**Comment by vmarkovtsev** [comment]
Yes, this is rooted in my experience with other testing frameworks where the arguments order is not Yoda-style. It would be nice to change everywhere, but the amount of required boring work has always stopped me.
